libs/wire-api/src/Wire/API/User.hs

```diff
@@ -892,6 +895,53 @@ instance ToSchema BindingNewTeamUser where
         <$> bnuTeam .= bindingNewTeamObjectSchema
         <*> bnuCurrency .= maybe_ (optField "currency" genericToSchema)
 
+--------------------------------------------------------------------------------
+-- SCIM User Info
+
+data ScimUserInfo = ScimUserInfo
+  { suiUserId :: UserId,
+    suiCreatedOn :: Maybe UTCTimeMillis
```

Comment:
How long would it take to add "createdby" to the csv file, while you're at it?  Seems like a good opportunity to avoid potential follow-up tickets, but I'm not sure.